src/main/scripts/src/app/shared/perspectives/documents/post-it/post-it-documents-perspective.component.ts

```diff
@@ -51,20 +53,24 @@ export class PostItDocumentsPerspectiveComponent implements Perspective, OnInit
   @Input()
   public query: string;
 
+  // TODO REMOVE
```

Comment:
When I finish the 'Edit attributes' and 'Delete documents' parts of the task. :)